app/scripts/modules/amazon/serverGroup/configure/wizard/location/basicSettings.html

```diff
@@ -96,12 +96,14 @@
           <label>
             <input type="checkbox"
                    ng-click="command.toggleSuspendedProcess('AddToLoadBalancer')"
-                   ng-checked="!command.processIsSuspended('AddToLoadBalancer')"/>
+                   ng-checked="!command.processIsSuspended('AddToLoadBalancer')"
+                   ng-disabled="command.strategy !== '' && command.strategy !== 'custom'"
+            />
             Send client requests to new instances
           </label>
         </div>
       </div>
-      <deployment-strategy-selector label-columns="3" field-columns="9" ng-if="!command.viewState.disableStrategySelection && command.selectedProvider" command="command"></deployment-strategy-selector>
+      <deployment-strategy-selector label-columns="3" field-columns="9" ng-if="!command.viewState.disableStrategySelection && command.selectedProvider" command="command" updateFn="enableProcess('AddToLoadBalancer')"></deployment-strategy-selector>
```

Comment:
I don't think this is doing anything - Angular 1 parses attributes as camel-cased, so it would need to be `update-fn`, unless the directive included an `updatefn` attribute, which it doesn't.